Program.cs

```diff
@@ -1,100 +1,116 @@
-                    }
-                    else
-                    {
-                        Console.WriteLine(options + " is not a valid option!");
-                    }
-                    break;
+                        // Giving the user options on casing.
+                        Console.WriteLine("\nDo you want your input to be upper or lower case?");
+                        Console.WriteLine("Indicate by typing (u) for upper or (l) for lower.\n");
+
+                        // The users option is stored in "options".
+                        string options = Console.ReadLine()/*?.ToLower()*/;
+
+                        // If the option is "u" uppercase, else lowercase.
+                        if (options == "u")
+                        {
+                            Console.WriteLine(input.ToUpper() + "\n");
+                        }
+                        else if (options == "l")
+                        {
+                            Console.WriteLine(input.ToLower() + "\n");
+                        }
+                        else
+                        {
+                            Console.WriteLine(options + " is not a valid option!");
+                        }
+                        break;
 
                     case 2:
-                    Console.WriteLine("\n");
-                    Console.WriteLine($"Today is {DateTime.Today:D}. It is a beautiful {DateTime.Now.DayOfWeek}." + "\n");
-                    break;
+                        Console.WriteLine($"\nToday is {DateTime.Today:D}. It is a beautiful {DateTime.Now.DayOfWeek}.\n");
+                        break;
 
                     case 3:
-                    Console.WriteLine("\n");
-                    // Requesting the user's input.
-                    Console.WriteLine("Please enter a number.");
-                    // The data is stored in "num1".
-                    int num1 = Convert.ToInt32(Console.ReadLine());
-                    // Requesting the user's input.
-                    Console.WriteLine("Please enter another number.");
-                    // The data is stored in "num1".
-                    int num2 = Convert.ToInt32(Console.ReadLine());
-                    // Requesting the user's input.
-                    Console.WriteLine("Please enter an arithmetic operator, either addition or subtraction.");
-                    // The operator is stored in "arithmetic"".
-                    var arithmetic = Console.ReadLine();
-                    Console.WriteLine("\n");
+                        // Requesting the user's input for the first number.
+                        Console.WriteLine("\nPlease enter a number.");
+                        // The data is stored in "num1".
+                        if (!int.TryParse(Console.ReadLine(), out int num1))
+                        {
+                            Console.WriteLine("Invalid input. Read please!");
+                            break;
```

Comment:
If you asre breaking the entire switch case ends so the next line will not run

Reply:
Yes it will. It will take the user back to the top of the code, to choose options.
Should I work on making the user input the correct format after being informed of the input being invalid?